src/cli/options/options.go

```diff
@@ -11,7 +11,10 @@ import (
 func Control() control.Options {
 	opt := control.Defaults()
 
-	opt.FailFast = fastFail
+	if fastFail {
+		opt.FailureHandling = control.FailFast
```

Comment:
What are our defaults for backup and restore?

For both it should be `failAfterRecovery`

Reply:
It is: see https://github.com/alcionai/corso/pull/3005/files#diff-8d31208fb3c9d64612a8181dd37f8031eaa1797a0a0c252829c939864df94ac5R32